all_test.go

```diff
@@ -1424,3 +1437,60 @@ func TestAuthArchiveIdentity(t *testing.T) {
 		t.Fatalf("TestArchiveIdentity failed, archived user should not be allowed to be rename identity: %v", err)
 	}
 }
+
+// Implemented in a smilar way to TestAuthUpdateDuplicateIdentity
+func TestAccountLocking(t *testing.T) {
+	c := setup(t)
+	defer Teardown(c)
+	c.MaxFailedLoginAttempts = 3
+	c.EnableAccountLocking = true
+
+	if isBackendPostgresTest() {
+		user, userErr := c.GetUserFromIdentity(joeEmail)
```

Comment:
This test is not clear enough. I'd expect to see a loop that iterates 4 times, trying to login, and receiving a InvalidPassword error the first 3 time, an then on the 4th time, getting back an AccountLocked error.
And then, on the outside of the loop, I'd expect another loop that switches between Lockable and NonLockable user policy. Basically, the test should be shorter and more concise, and more obvious.